examples/013_thermal_example.py

```diff
@@ -26,15 +26,17 @@
 Thermal analysis
 ----------------
 
-PyDPF Composites can also be used to post-process thermal analyses.
-In this case, the simulation is a two-step analysis where the results of
-a thermal analysis are an input of the structural analysis. So, the RST
-contains temperature and structural results.
-The example mimics a PCB which was modeled with Ansys Composites PrePost (ACP).
-where the solid model feature of ACP is used to generate the volume mesh.
+This example shows how to evaluate a thermal analysis.
+The simulation is a two-step analysis where the results of
+a thermal analysis are an input of the structural analysis.
```

Comment:
@roosre Can you clarify what this sentence means?

Here is my suggested revision (but I don't know if it's technically correct):
```suggestion
The simulation's thermal analysis results are also
the input of a structural analysis.
```

I've revised the rest of the example!